application/todo/src/main/java/jp/kztproject/rewardedtodo/application/reward/GetTodoListInteractor.kt

```diff
@@ -1,8 +1,8 @@
 package jp.kztproject.rewardedtodo.application.reward
 
+import jp.kztproject.rewardedtodo.domain.todo.Todo
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/kseito/RewardedTodo/blob/6955c46b72f59d8c61a7d2312ea70f75ea4fc4e7/application/todo/src/main/java/jp/kztproject/rewardedtodo/application/reward/GetTodoListInteractor.kt#L3"></span>Imports must be ordered in lexicographic order without any empty lines in-between</td>
    </tr>
  </tbody>
</table>